webextensions/api/webRequest.json

```diff
@@ -3799,6 +3967,27 @@
                 }
               }
             },
+            "thirdParty": {
```

Comment:
@bershanskiy This should be after `tabId`.

Reply:
Thanks for catching this, I'll fix it.

(I missed this because I ran `npm run fix` and expected it to catch all formatting details. I'll make sure to manually check property orders in future PRs.)

Reply:
Huh, this is odd...  The linter should be catching this as an issue, but I’m looking at the output and it’s apparently accepting this.  Can you do me a favor and merge the latest `master` as well?

Reply:
@vinyldarkscratch I merged `master`, linter is still silent.

Reply:
That's really odd...  If you perform the move, does the linter raise an error?

Reply:
I fixed the order and linter did not twitch.

Reply:
That's not right at all...  I'll add this to my agenda and see if I can get a fix for it!